package.json

```diff
@@ -102,7 +102,7 @@
     "api-watch": "nodemon -e \".ts\" --exec npm run babel-node-api",
     "babel-node-api": "babel-node -r dotenv/config -x \".ts\" ./src/api/api.ts",
     "build-api": "npx tsc --build ./tsconfig-api.json",
-    "test-api": "npx jest --testTimeout=5000 --config=./src/api/jest.config.ts",
+    "test-api": "npx jest --testTimeout=5000 --forceExit --runInBand --config=./src/api/jest.config.ts",
```

Comment:
What does using [forceExit ](https://jestjs.io/docs/cli#--forceexit)actually achieve here? The docs suggest this is an escape-hatch but that resources should be torn down after each test instead so that Jest can shut down cleanly.

Reply:
It allows the test to finish. Lionell and I couldn't find the underlying issue, may be related to supertest, and there was already a comment in our code recommending this.